db/migrate/20220808200759_add_datacite_xml_to_work.rb

```diff
@@ -0,0 +1,5 @@
+class AddDataciteXmlToWork < ActiveRecord::Migration[6.1]
+  def change
+    add_column :works, :datacite_xml, :text
```

Comment:
👍 Assuming we will delete the other column after the full refactor

Reply:
Yes, that was my plan.